modules/bitgo/example/js/btc/check-lightning-balance.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Check the lightning balance of a wallet.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
```

Comment:
How about
```suggestion
const coin = env === 'prod' ? 'btc' : 'tbtc';
```

Reply:
Added `const coin = env === 'test' ? 'tbtc' : 'btc';` because TypeScript complains the other way round.